cmd/clusterctl/cmd/rollout/restart.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rollout
```

Comment:
I noticed that in the `clusterctl/cmd` directory, we have the following additions:
```shell
# Deleted some lines to reduce verbosity.
├── cmd
│   ├── alpha.go
│   ├── config.go
│   ├── config_cluster.go
│   ├── config_provider.go
│   ├── config_repositories.go
│   ├── config_repositories_test.go
│   ├── generate.go
│   ├── generate_yaml.go
│   ├── generate_yaml_test.go
│   ├── get.go
│   ├── get_kubeconfig.go
│   ├── rollout
│   │   └── restart.go
│   ├── rollout.go
│   ├── upgrade.go
│   ├── upgrade_apply.go
│   ├── upgrade_plan.go
...
```

Regarding structure, we have been naming subcommands as defined by `command_subcommand.go`. It seems that if we strictly follow the convention we will have `alpha_rollout_restart.go`. I can see that this may be contentious in the future if we wanted to "graduate" the command from an alpha command into a standard command. So as a compromise, I suggest doing something like this.
```shell
# Deleted some lines to reduce verbosity.
├── cmd
│   ├── alpha.go
│   ├── rollout.go
│   ├── rollout_restart.go
│   ├── rollout_status.go
│   ├── rollout_undo.go
...
```
This way in the future, if we need to "graduate" the rollout cmd from alpha, it will just involve removing it as a subcommand from the `alpha.go` file and no other changes.
WDYT?

Reply:
I had it that way initially, but didn't like the explosion in files in that `/cmd` that this would lead to. So, decided to create a separate directory for each sub-command of `rollout`.

Reply:
My only concern here is code consistency. I don't necessarily mind that we have these commands in a subdirectory, it's just that we now have some commands in a subdirectory vs the other in `cmd_subcmd.go` format.

Again, not a major concern but maybe @fabriziopandini can weigh in.